h2o-r/tests/testdir_misc/runit_connect.R

```diff
@@ -0,0 +1,31 @@
+setwd(normalizePath(dirname(
+  R.utils::commandArgs(asValues = TRUE)$"f"
+)))
+
+h2oRDir <- normalizePath(paste(dirname(R.utils::commandArgs(asValues=TRUE)$"f"),"..","..",sep=.Platform$file.sep))
+to_src <- c("aggregator.R", "classes.R", "connection.R","config.R", "constants.R", "logging.R", "communication.R",
+            "kvstore.R", "frame.R", "targetencoder.R", "astfun.R","automl.R", "import.R", "parse.R", "export.R", "models.R", "edicts.R",
+            "coxph.R", "coxphutils.R", "gbm.R", "glm.R", "gam.R", "glrm.R", "kmeans.R", "deeplearning.R", "randomforest.R", "generic.R",
+            "naivebayes.R", "pca.R", "svd.R", "locate.R", "grid.R", "word2vec.R", "w2vutils.R", "stackedensemble.R", "rulefit.R",
+            "predict.R", "xgboost.R", "isolationforest.R", "psvm.R", "segment.R", "tf-idf.R", "explain.R", "permutation_varimp.R", "extendedisolationforest.R")
+src_path <- paste(h2oRDir,"h2o-package","R",sep=.Platform$file.sep)
+invisible(lapply(to_src,function(x){source(paste(src_path, x, sep = .Platform$file.sep))}))
+
+source("../runitUtils/utilsR.R")
+default.packages()
+Log.info("Loaded default packages. Additional required packages must be loaded explicitly.")
```

Comment:
```suggestion
source("../../scripts/h2o-r-test-setup.R")
```
this should be enough.
Did you have any problem with the approach used in other tests?

Reply:
Yes, `source("../../scripts/h2o-r-test-setup.R")` seems like it expects h2o to be running in the background.